Firestore/core/src/firebase/firestore/util/hard_assert.h

```diff
@@ -119,20 +116,13 @@ typedef void (*FailureHandlerCallback)(const char* file,
  *     not expected to return. (If it does, std::terminate() will be called
  *     immediately after it does so.)
  */
-void SetFailureHandler(FailureHandlerCallback callback);
+void SetFailureHandler(FailureHandler callback);
 
 /**
- * Default failure handler for non-apple platforms. This should *typically* not
- * be called directly, though could be called as an implementation detail of a
- * custom failure handler. eg a failure handler that added logging might look
- * like:
- *
- *   SetFailureHandler([](...) {
- *     Log("There was an error");
- *     DefaultFailureHandlerCallback(...);
- *   });
+ * Default failure handler for non-apple platforms. This should typically not
```

Comment:
This is just the default failure handler. It's even the default no Apple platforms when using just C++.